tests/optimization/test_optimization.py

```diff
@@ -166,5 +166,21 @@ def test_schedulers(self):
             )
 
             scheduler = scheduler_func(self.optimizer, **kwargs)
+            if scheduler_func.__name__ != "get_constant_schedule":
+                LambdaScheduleWrapper.wrap_scheduler(scheduler)  # wrap to test picklability of the schedule
```

Comment:
Thank you for the comments and sorry for the delay

Just wanted to call out here that I don't love this conditional, but it's the alternative to doing the unneeded `_get_constant_schedule` partial to get the test to pass. If you prefer a different approach just lmk